views/404.ejs

```diff
@@ -0,0 +1,8 @@
+<%- include('includes/head.ejs') %>
```

Comment:
the directory name should be more generic, like: 'files' or 'headers' or something.

Reply:
Ejs doesn't support layout , so we need to categorizes the common html code that are repeated in many view pages . it may contains header , footer , navigation bar , side view . Give me some good name for it .